src/analytics/Events.tsx

```diff
@@ -42,6 +42,7 @@ export enum HomeEvents {
   view_token_balances = 'view_token_balances',
   view_nft_home_assets = 'view_nft_home_assets', // When "NFTs" is clicked in Home Assets Pages
   home_action_pressed = 'home_action_pressed',
+  notification_center_pressed = 'notification_center_pressed',
```

Comment:
Should we name it more precisely? Anticipating the case where there's several ways to reach the notification center and we may want to know the exact button that triggered it.
```suggestion
  notification_bell_pressed = 'notification_bell_pressed',
```